### ISSUE TITLE
DDL: Standardize spaces around colon, coma... with namespaces

### ISSUE THREAD

**Issue by jrodriguez-rc** [SAP/abap-cleaner#359]
This rule is adding spaces around the slashes of namespaces, converting this
```
define root view entity /RHP/C_DocumentDefTP
  provider contract transactional_query
  as projection on /RHP/R_DocumentDefTP
```
into
```
define root view entity / RHP / C_DocumentDefTP
  provider contract transactional_query
  as projection on / RHP / R_DocumentDefTP
```

**Comment by jmgrassau** [comment]
Hi Jaime, 

that's not good… thanks for reporting this, will fix asap! 

Kind regards,
Jörg-Michael

**Comment by jmgrassau** [comment]
Hi Jaime, 

thanks again for reporting this bug so quickly – this should now be fixed with version 1.19.2, which was just released! I tried to create some extreme (but syntactically correct) examples to ensure that the distinction between using ```/``` as an arithmetic operator and using it for a namespace (or between using ```-``` as a sign and using it as an operator) is now correctly made: 

![image](https://github.com/user-attachments/assets/fc24e635-e0e5-4bcd-8835-42403ee24b03)

Kind regards,
Jörg-Michael

**Comment by jrodriguez-rc** [comment]
Thank you for the quick fix!
